src/viur/core/skeleton.py

```diff
@@ -1191,14 +1187,14 @@ def txnDelete(skel: SkeletonInstance):
                 bone.delete(skel, boneName)
                 if bone.unique:
                     flushList = []
-                    for lockValue in viurData.get("%s_uniqueIndexValue" % boneName) or []:
-                        lockKey = db.Key("%s_%s_uniquePropertyIndex" % (skel.kindName, boneName), lockValue)
+                    for lockValue in viurData.get(f"{boneName}_uniqueIndexValue") or []:
+                        lockKey = db.Key(f"{skel.kindName}_{boneName}_uniquePropertyIndex", lockValue)
                         lockObj = db.Get(lockKey)
                         if not lockObj:
-                            logging.error("Programming error detected: Lockobj %s missing!" % lockKey)
+                            logging.error(f"Programming error detected: Lockobj {lockKey} missing!")
                         elif lockObj["references"] != dbObj.key.id_or_name:
                             logging.error(
-                                "Programming error detected: %s did not hold lock for %s" % (skel["key"], lockKey))
+                                f"""Programming error detected: {skel["key"]} did not hold lock for {lockKey}""")
```

Comment:
Oh really, a Programming error detected??? wtf... 
```suggestion
                                f"""{skel["key"]!r} does not hold lock for {lockKey!r}""")
```